spec/models/form/sales/subsections/setup_spec.rb

```diff
@@ -49,6 +50,7 @@
   context "when start year is >= 2024" do
     before do
       allow(section.form).to receive(:start_year_2024_or_later?).and_return(true)
+      allow(section.form).to receive(:start_year_2025_or_later?).and_return(false)
```

Comment:
It would be worth adjusting the context name, since this is now insisting that the start year is 2024.

Is there anything in these blocks it's worth separately testing for 2025 onwards? (There may not be, I've not checked)